File: codes/PriorityQueue/Python/PriorityQueueMain.py
import PriorityQueueModule as pqm

pq = pqm.PriorityQueue(lambda ch1, ch2: ord(ch2)-ord(ch1))

pq.PEnqueue('A')
pq.PEnqueue('B')
pq.PEnqueue('C')
print(pq.PDequeue())

pq.PEnqueue('A')
pq.PEnqueue('B')
pq.PEnqueue('C')
print(pq.PDequeue())

while pq.PQIsEmpty() is not True:
    print(pq.PDequeue())

File: codes/PriorityQueue/Java/PriorityQueue/PriorityQueueMain.java
package PriorityQueue;

public class PriorityQueueMain {
    public static void main(String[] args) {
        PriorityQueue<Character> pq = new PriorityQueue<Character>((ch1, ch2)->ch2-ch1);

        pq.PEnqueue('A');
        pq.PEnqueue('B');
        pq.PEnqueue('C');
        System.out.println(pq.PDequeue());

        pq.PEnqueue('A');
        pq.PEnqueue('B');
        pq.PEnqueue('C');
        System.out.println(pq.PDequeue());

        while(!pq.PQIsEmpty())
            System.out.println(pq.PDequeue());
    }
}
